src/core/stream/orchestrator/active_period_emitter.ts

```diff
@@ -101,6 +101,8 @@ export default function ActivePeriodEmitter(
             return acc;
           }
         }
+        // /!\ fallthrough done one purpose
```

Comment:
Do you think it is possible to explain why ?

Reply:
Yes.
I've also written "one" instead of "on"!

Reply:
I was wondering if i was that bad for understanding English. Now I am relieved.

Reply:
done